src/components/generic-card/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { SpacingProp } from "@/types/style";
+import { H2, P } from "../text";
+import s from "./GenericCard.module.css";
+
+interface GenericCardProps {
+  title: string;
+  description: string;
+  padding?: SpacingProp;
+  children?: React.ReactNode;
+}
+
+export default function GenericCard({
+  title,
+  padding = "56px",
+  description,
+  children,
+}: GenericCardProps) {
+  return (
+    <div className={s.genericCard} style={{ padding }}>
+      <H2 className={s.title}>{title}</H2>
+      <P className={s.description}>{description}</P>
+      <div className={s.buttons}>{children}</div>
```

Comment:
I think naming this `buttons` assumes the content too specifically now (I think we will use it for other things, right?)

Reply:
yeah, Seems like a better change, have updated now to be less generic and went with `children` although it may need to be remove later on and have the styling dictated by the child component.

Also added this as a conditional render so we don't have empty divs floating about

Reply:
Github might be having a moment though 🥴 
<img width="274" alt="Screenshot 2024-12-26 at 21 46 02" src="https://github.com/user-attachments/assets/e7663198-832a-427b-a44a-c1d42d92ecc7" />

Reply:
Haha all good; I'll merge this when it comes through!